src/jit/error.cpp

```diff
@@ -389,13 +389,19 @@ void logf(const char* fmt, ...)
     // If it fails to log an LL_INFO1000 message once 
     // it will always fail when logging an LL_INFO1000 message.
     //
-    if (logToEEfailed || !vlogf(LL_INFO1000, fmt, args))
+    if (logToEEfailed)
```

Comment:
Similar here:

    if (!logToEEfailed)
    {
        va_start(args, fmt);
        if (!vlogf(LL_INFO1000, fmt, args))
            logToEEfailed = true;
        va_end(args);
    }

    if (logToEEfailed)
    {
        // If the EE refuses to log it, we try to send it to stdout
        va_start(args, fmt);
        logf_stdout(fmt, args);
        va_end(args);
    }